.eslintrc.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
I think it should be easy to pull in indeed's eslint rules, so we might as well do that as part of this PR. If they haven't already been merged into starfish (as eslintrc.json, not eslintrc.js), then you can find the .json file in the starfish PR that Dani put up this morning.

Reply:
And I think you'll want to include the typescript stuff Indeed uses too.
You can find it in this file https://code.corp.indeed.com/frontend/eslint-plugin-indeed/blob/master/settings.js
There's the stuff in TYPESCRIPT_BASE (though we don't need `browser: true` and I don't know if we need `parser: 'babel-eslint' or `sourceType: 'module'`
There's also the stuff in OVERRIDES.TYPESCRIPT
You might need to make sure none of those rules conflict with the rules in other parts of our eslintrc. I believe Indeed uses 'overrides' because they can choose which set or sets of rules to use. So it's possilbe that some of them conflict? I don't know, I haven't looked at it closely - you might want to.

Reply:
Thanks Dani, at the moment I'm gettiing the eslint / prettier stuff in, where there is conflict of course. The typescript stuff will probably be another task.

Reply:
What I meant when I said 'the typsecript stuff' was 'the typescript eslint rules' so if you're trying to put in all of Indeed's eslint rules now, you'll want to look at the link I posted and add them now. 
Only copying the eslint rules I put into Starfish isn't enough because Starfish isn't currently using typescript, so I didn't include those rules in its eslintrc.
If adding Indeed's typescript rules is another task for later, that's cool. Just making sure I'm being clear because I thought you were trying to add all of Indeed's eslint rules now.